vault/resource_jwt_auth_backend_test.go

```diff
@@ -72,6 +72,61 @@ func TestAccJWTAuthBackend_OIDC(t *testing.T) {
 	})
 }
 
+// TODO: OIDC plugin has a validation stage when configs are written,
+// and since we use circleci Docker, we can't mount files nor can it see a
+// local one. This will need to be updated when we switch to a Circle machine
+// executor.
+
+// The random numbers on the provider_config are a hash of the object.
+func TestAccJWTAuthBackend_OIDC_Provider_ConfigAzure(t *testing.T) {
+	path := acctest.RandomWithPrefix("oidc")
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testJWTLocal(t) },
+		Providers:    testProviders,
+		CheckDestroy: testJWTAuthBackend_Destroyed(path),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccJWTAuthBackendConfigOIDCProviderConfigAzure(path),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("vault_jwt_auth_backend.azure", "oidc_discovery_url", "https://accounts.google.com"),
+					resource.TestCheckResourceAttr("vault_jwt_auth_backend.azure", "provider_config.#", "1"),
+					resource.TestCheckResourceAttr("vault_jwt_auth_backend.azure", "provider_config.2995719985.provider", "azure"),
+				),
+			},
+		},
+	})
+}
+
+// TODO: OIDC plugin has a validation stage when configs are written,
+// and since we use circleci Docker, we can't mount files nor can it see a
+// local one. This will need to be updated when we switch to a Circle machine
+// executor.
+
+// The random numbers on the provider_config are a hash of the object.
+func TestAccJWTAuthBackend_OIDC_Provider_ConfigGSuite(t *testing.T) {
+	path := acctest.RandomWithPrefix("oidc")
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testJWTLocal(t) },
+		Providers:    testProviders,
+		CheckDestroy: testJWTAuthBackend_Destroyed(path),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccJWTAuthBackendConfigOIDCProviderConfigGSuite(path),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("vault_jwt_auth_backend.gsuite", "oidc_discovery_url", "https://accounts.google.com"),
+					resource.TestCheckResourceAttr("vault_jwt_auth_backend.gsuite", "provider_config.1247099397.provider", "gsuite"),
+					resource.TestCheckResourceAttr("vault_jwt_auth_backend.gsuite", "provider_config.1247099397.gsuite_service_account", "/tmp/service-account.json"),
```

Comment:
How about reading the path to the service account file from an env variable, so that it isn't hardcoded to one location? That could even be the condition for skipping this test, similar to what's done elsewhere for MYSQL_URL: https://github.com/hashicorp/terraform-provider-vault/blob/3e79768a0c205ae6788653ca16e0fb3034e4f0d0/vault/resource_database_secret_backend_connection_test.go#L241-L244

Reply:
Done!